docs/graasp-insights-user-guide.md

```diff
@@ -0,0 +1,243 @@
+
+![Dataset screen](./figures/dataset-screen.png)
+
+If the dataset was loaded as a `csv` or `xlsx` file, or if the structure of the dataset supports it, you can switch to a `TABLE` view to display a tabular version of it.
+
+![Dataset screen (table view)](./figures/dataset-screen-table.png)
+
+# Algorithms
+
+The `Algorithms` tab contains every algorithm that can be used throughout Graasp Insights. It is also the entry point for accessing, editing and writing algorithms.
+
+![Algorithms tab](./figures/algorithms.png)
+
+## Edit algorithm
+
+Each algorithm can be viewed and edited with the `Edit Algorithm` button.
+
+![Edit algorithm](./figures/edit-algorithm.png)
+
+An editor shows up on the left for you to modify the code of the algorithm. Additionally, you can modify the name, description, type and parameters on the right.
+
+For more information about how to write an algorithm within Graasp Insights, jump to the [Adding an algorithm](#Adding-an-algorithm) section.
+
+When you have finished editing the algorithm click on the `Save` button to save your modifications. Alternatively, click on the `Back` button to discard your changes and go back to the `Algorithms` screen.
+
+# Executions
+
+Use the `Executions` tab to execute algorithms on datasets.
+
+Select a dataset, the algorithm you want to apply, and (optionally) a name for the resulting dataset.
+
+![Executions tab](./figures/executions.png)
+
+If the algorithm has parameters, then a green `EDIT PARAMETERS` button should appear for you to edit them.
+
+![Executions tab](./figures/executions-edit-parameters.png)
+
+> Note: If there is a field selection (as in the example above) and the schema doesn't correspond to the dataset, you can generate a new schema for your dataset in the `Schemas` tab.
+
+Finally, start the execution by clicking on the `EXECUTE` button.
+
+The status of your new execution can then be tracked with the table underneath. A green "check" icon will indicate that the execution has finished and was successful. A red cross will indicate that an error has happened.
+
+![Executions tab with table](./figures/executions-with-table.png)
+
+The Dataset select menu also includes the resulting datasets. This allows you to chain and apply multiple algorithms to a dataset.
+
+> Note: you can also check the version of your python installation by hovering on the top right python logo. The logo turns gray if no python installation is found.
+
+Click on the `View execution` icon to see more information about the execution. From there, the used parameters and the output log are also displayed. The output log is particularly useful for debugging your algorithm if an error happened.
+
+![Execution view](./figures/execution-view.png)
+
+# Validations
+
+Use the `Validations` tab to execute validation algorithms that test whether datasets have been correctly anonymized.
+
+![Validations tab](./figures/validations.png)
+
+Click on the lower-right `+` button to validate a dataset.
```

Comment:
Maybe: _Click on the lower-right `+` button to begin validation_ OR _...to enter the validation workflow_